api/services/TemplateResolver.js

```diff
@@ -0,0 +1,15 @@
+const templates = require('../../config/templates');
```

Comment:
This is kind of overkill but it makes testing easier, and it doesn't rely on us having templates in the`config/templates` file that match our edge cases.